hpat/tests_perf/test_perf_unicode.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+import unittest
+
+from numba import njit, types
+
+import hpat
```

Comment:
Looks like hpat is used nowhere. You only use njit from numba, but not jit from hpat. So I think we need either to remove the import or to use hpat in the code.

Reply:
We do not need tests in HPAT that do not touch HPAT.